### ISSUE TITLE
[Feature Request] Update for Water Picture Status Card

### ISSUE THREAD

**Issue by usedjeans818** [ollo69/ha-smartthinq-sensors#131]
**Is your feature request related to a problem? Please describe.**
There is a mistake in the Lovelace "cards" code.  I've also created new image files that build on the washerrunning.gif file.  (Imagine the washerrunning.gif animation running behind an image of both a washer and a dryer.)

**Describe the solution you'd like**
The ```not_state: "on"``` should be ```state_not: "on"```

**Describe alternatives you've considered**
I would like to submit new images as well as updated code for both a washer and dryer.  Please let me know how I can provide the updated image files.  I don't have permissions to upload them to the [/washerpics/](https://github.com/ollo69/ha-smartthinq-sensors/tree/master/washerpics) page.

**Additional context**

**Washer Sensor:**
```
sensor:
  - platform: template
    sensors:
      washer_cycle_state:
        value_template: '{{state_attr(''sensor.my_washer'', ''remain_time'')}}'
        friendly_name: Washer Cycle State
        icon_template: 'mdi:washing-machine'
```
**Washer Lovelace:**
```
cards:
  - type: conditional
    conditions:
      - entity: sensor.my_washer
        state: "on"
    card:
      aspect_ratio: '0.70'
      entity: sensor.washer_cycle_state
      image: /local/washer_on.gif
      type: picture-entity
  - type: conditional
    conditions:
      - entity: sensor.my_washer
        state_not: "on"
    card:
      aspect_ratio: '0.70'
      entity: sensor.my_washer
      image: /local/washer_off.jpg
      type: picture-entity
type: vertical-stack
```

**Dryer Sensor:**
```
sensor:
  - platform: template
    sensors:
      dryer_cycle_state:
        value_template: '{{state_attr(''sensor.my_dryer'', ''remain_time'')}}'
        friendly_name: Dryer Cycle State
        icon_template: 'mdi:tumble-dryer'
```
**Dryer Lovelace:**
```
cards:
  - type: conditional
    conditions:
      - entity: sensor.my_dryer
        state: "on"
    card:
      aspect_ratio: '0.70'
      entity: sensor.dryer_cycle_state
      image: /local/dryer_on.gif
      type: picture-entity
  - type: conditional
    conditions:
      - entity: sensor.my_dryer
        state_not: "on"
    card:
      aspect_ratio: '0.70'
      entity: sensor.my_dryer
      image: /local/dryer_off.jpg
      type: picture-entity
type: vertical-stack
```

**Comment by KTibow** [comment]
Send a PR.

**Comment by usedjeans818** [comment]
Sorry, @KTibow , but what's a PR?

**Comment by KTibow** [comment]
It's a pull request. Whenever you don't know an acronym, google it.

**Comment by usedjeans818** [comment]
Thanks for the education as I'm not really a programmer and have not used GitHub in any capacity other than having submitted maybe two issues (I did Google "PR," but maybe should've Googled "pr github" instead).

I took a quick look at Pull Requests and it's not as straightforward as I would've hoped.  Is this still a Pull Request given that I'm not proposing any changes to the actual "ha-smartthinq-sensors" code and am only suggesting changes to the write-up at the bottom?
